### ISSUE TITLE
Regressions with drag and dropping images in 0.8.6

### ISSUE THREAD

**Issue by lylo** [KonnorRogers/rhino-editor#135]
I've been trying the 0.8.6 release and I've found a couple of regressions, one minor, one major.

**Minor**

Adding an image and clicking on the caption clears the placeholder (nice!) but the cursor is in the wrong position. In Firefox it's position too high (see video), in Chrome it kinda disappears. Would be nice if the cursor appeared in the centre of the edit field.

**Major**

Dragging one image is fine. Dragging a second doesn't work. There's no error, it's just... consumed somewhere and not added to the DOM. Oddly, if you add some text after the first image, dragging a second image works fine. See video below.

https://github.com/KonnorRogers/rhino-editor/assets/5447/dcc5a3d9-d02e-45cc-b996-6d578e38d381

**Comment by KonnorRogers** [comment]
@lylo hmmm...interesting...probably has to do with the schema / gallery changes. I'll take a look! Thanks for reporting these, really hard to go through all the permutations...I'll work on a regression test for the drag and drop images as well. Sorry about that.

**Comment by lylo** [comment]
No need for apologies @KonnorRogers, I feel bad about reporting 😅  I'm using it it anger every day at the moment so I'll file anything else that comes up, hope that's ok. It's mainly the attachments and galleries that I find problems with, haven't spotted anything untoward with general text editing as yet.

**Comment by KonnorRogers** [comment]
Attachments and galleries have been the biggest headache without a doubt 🙈 

Please do keep filing! I appreciate it a ton!

**Comment by lylo** [comment]
> @lylo I can't seem to reproduce the cursor issue for the caption. I tested in all 3 browsers 🤔

Gah, this is my bad. I had an override going on in my CSS which was breaking it. Sorry about that. Removing those it now looks a lot better. Maybe could do with a slight bit more top margin, unless there's something else going on with my set up?

<img width="216" alt="Screenshot 2023-09-26 at 10 13 21" src="https://github.com/KonnorRogers/rhino-editor/assets/5447/3551fa2b-6d41-491a-bb35-5c8825f5ba0e">

One other point of note is that I'm using Import Maps. This is fine but you can't include CSS in your application.js file when taking this approach:

```
import "rhino-editor/exports/styles/trix.css"
```

So for now I'm including this in my layout:

```
    <link href="https://unpkg.com/rhino-editor/exports/styles/trix.css" rel="stylesheet">
```

Maybe this is another source of problems? I should probably try and vendor it shouldn't I?

**Comment by KonnorRogers** [comment]
@lylo that's expected. I need to update docs to talk about importmaps, just haven't gotten around to it. That's the expected way to use CSS for importmaps. I do also think there should be some additional top margin for those figcaptions. That is correct tho.

**Comment by KonnorRogers** [comment]
@lylo I believe this is fixed in 0.8.7. Feel free to re-open if not.
